src/components/DayCard/DayCard.tsx

```diff
@@ -16,10 +16,16 @@ export const DayCard: React.FC<IProps> = (props: IProps): React.ReactElement | n
 
   return (
     <s.Container>
-      { isDayPicture(props.dayPicture) ? (
+      {isDayPicture(props.dayPicture) ? (
         <>
           <s.Date>{readableDate(props.dayPicture.date)}</s.Date>
-          <s.DayImageContainer url={props.dayPicture.url} />
+          {props.dayPicture.media_type === 'image' ? (
+            <s.DayImageContainer url={props.dayPicture.url} />
+          ) : (
+            <a href={props.dayPicture.url}>
```

Comment:
Maybe better to open a new tab?!
https://stackoverflow.com/questions/15551779/open-link-in-new-tab-or-window